share/completions/qmk.fish

```diff
@@ -0,0 +1,12 @@
+function __fish_qmk_complete
```

Comment:
Since we now have this twice, I would prefer if it was extracted into a function, named `__fish_argcomplete_complete` for example, that just takes the commandname.

So you'd make a file by that name in share/functions/, add the function, and then just call

```fish
__fish_argcomplete_complete qmk
```

here and

```fish
__fish_argcomplete_complete az
```

in the az completions. The function would call

```fish
$argv 8>&1 9>&2 2>/dev/null
```


That way, adding more of these is just a matter of adding one line into a new file.

Reply:
done